lstmcpipe/stages/mc_process_dl1.py

```diff
@@ -543,17 +543,15 @@ def submit_dl1_jobs(
         files = [f.as_posix() for f in Path(dir_lists).glob("*")]
 
         slurm_options = {}
-        slurm_options["output"] = os.path.join(
-            JOB_LOGS, f"job_%A_%a_{'train' if set_type=='training' else 'test'}.o"
-        )
-        slurm_options["error"] = os.path.join(
-            JOB_LOGS, f"job_%A_%a_{'train' if set_type=='training' else 'test'}.e"
-        )
+        slurm_options["output"] = \
+            job_logs_dir.joinpath(f"job_%A_%a_{'train' if set_type=='training' else 'test'}.o")
+        slurm_options["error"] = \
+            job_logs_dir.joinpath(f"job_%A_%a_{'train' if set_type=='training' else 'test'}.e")
         if particle == "proton":
```

Comment:
This can probably be removed unless we want to put them into long again.
But I dont think we need to

Reply:
Tricky point, not yet "optimised". Indeed it depends on the number of jobs that each `process_dl1` job contains. Also on the kind of workflow that we used (lstchain, hiperta, ctapipe).
Protons usually take longer, indeed I would leave them on long by default just in case.

Reply:
not always 🙈 , but indeed you're completely right.
Opening an issue to optimise this :)